recipes/iio-emu/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "iio-emu" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/analogdevicesinc/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 829286862f1f00c16996ff52700948018e8aee5706888daa1474834bc9c8c34a
+  patches:
+  - 0001-Add-algorithm-headers-for-min-max-on-Windows.patch  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+    - libtinyiiod
+    - libxml2
+  host:
+    - libtinyiiod
+    - libxml2
+    - libiio
+
+
+test:
+  commands:
+    # verify that the tools get installed
+    - test -f $PREFIX/bin/iio-emu  # [not win]
+    - test -f $PREFIX/bin/xml_gen  # [not win]
+    - if not exist %PREFIX%\\Library\\bin\\iio-emu.exe exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\bin\\xml_gen.exe exit 1  # [win]
```

Comment:
You should add a test that runs the help menu or version of the tools to ensure they link correctly at runtime.